packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js

```diff
@@ -97,6 +97,9 @@ const getMediaGalleryEntries = (product, optionCodes, optionSelections) => {
 // product has multiple related categories. This function filters and selects
 // one category id for that purpose.
 const getBreadcrumbCategoryId = categories => {
+    if (!!categories.length === false) {
```

Comment:
I'm guessing `categories` is an empty array when we get here and there is no category assigned to a product. In that case, let's just use a simple check against truthyness of `categories` and check whether there are any values in `categories` array, if it exists:

```js
// Exit if there are no categories for this product.
if (!categories || !categories.length) {
  return;
}
```

Reply:
@andrewbess Thanks for contributing! Looks like a simple fix. 👍 

@sirugh Agreed, good suggestion.

Reply:
Hello @sirugh 
Thank you for review.
I have fixed it.
Please check.